website/docs/add-secure-apps/providers/radius/HashSupport.tsx

```diff
@@ -0,0 +1,82 @@
+import styles from "./styles.module.css";
```

Comment:
Ported from #14119 -- Removes import on react-feather and detangles the dependency logjam.